README.md

```diff
@@ -1,61 +1,53 @@
 # ansible-aci
 
-The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
+## Description
 
-This collection has been tested and supports ACI 3.2+.
-Modules supporting new features introduced in ACI API in specific ACI versions might not be supported in earlier ACI releases.
+The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
 
-*Note: This collection is not compatible with versions of Ansible before v2.8.*
+See the [Cisco.Aci collection index](https://galaxy.ansible.com/ui/repo/published/cisco/aci/content/) for a full list of modules and plugins.
 
 ## Requirements
 
-Ansible v2.14 or newer
+- Ansible v2.15 or newer
+- Python v3.10 or newer
 
-## Install
+*Note: This collection is not compatible with versions of Ansible before v2.8.*
 
-Ansible must be installed
+Follow the [Installing Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) guide for detailed instructions.
 
-```sh
-sudo pip install ansible
-```
+## Installation
 
-Install the collection
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
 
 ```sh
 ansible-galaxy collection install cisco.aci
 ```
 
-## Use
+You can also include it in a requirements.yml file and install it with ansible-galaxy collection install -r requirements.yml, using the format:
```

Comment:
put the cli into a quote block with sh type like above

Reply:
Done